### DISCUSSION TITLE
تنظیمات uTLS

### DISCUSSION THREAD

**Discussion by BehnamAB** [hiddify/Hiddify-Manager#253] [Q&A سوالات رایج]
سلام چطور میتونم کاری کنم که توتنظیمات کانفیگ uTLS روی chrome قرار نگیره ؟

**Comment by robin98** [comment]
تا الان که آپشنی از محیط ui براش نذاشتن. چرا میخوایید همچین کاری کنید چونکه عملا فرق خاصی نداره و مساله فقط فعال بودن uTLS هست.
اگه خیلی تمایل به اینکار دارید به نظرم دستی باید برید و فایل های کانفیگ hiddify و اونایی که آخرشون .template دارند رو پیدا و ویرایش کنید، که من تخصصی در این زمینه ندارم. فقط یه ایده بود.

**Reply by hiddify-com** [reply]
اضافه شد :دی

**Reply by hosseingh1993** [reply]
توی ios نمیشه utls زد؟

**Reply by ghost** [reply]
> اضافه شد :دی
damet garm eyval 😍 ishalla droplist alpn ba h2 support ham bia tangesh 👯

**Comment by hosseingh1993** [comment]
توی ios هیچ جور نمیشه utls فعال کرد؟

**Reply by R30na** [reply]
توی اپ nV باید دستی بری تغییرش بدی و اگه میخوای کانفیگ به کسی بدی باید json بدی وارد کنه

**Reply by hosseingh1993** [reply]
کجاشه داداش؟
ادیت که میزنی کجاس؟> توی اپ nV باید دستی بری تغییرش بدی و اگه میخوای کانفیگ به کسی بدی باید json بدی وارد کنه

**Reply by R30na** [reply]
زیر توضیح دادم👇

**Comment by R30na** [comment]
از اینجا میتونی ادیت کنی : 

![IMG_0837](https://user-images.githubusercontent.com/13730082/222008462-35e9b6dd-e424-4b14-b179-3aa40efab79d.jpg)

برای وارد کردن json  هم میتونی از این گزینه استفاده کنی :

![IMG_0838](https://user-images.githubusercontent.com/13730082/222008528-fc529717-e9f0-46a9-bb21-c4703f7fe24c.jpg)

**Reply by R30na** [reply]
اینم یه نمونه json  که میتونی استفاده کنی برای vless+grpc البته فقط روی ایرانسل جواب میده: 

```json
{
  "routing": {
    "domainStrategy": "Asls"
  },
  "inbounds": [
    {
      "sniffing": {
        "enabled": false
      },
      "listen": "127.0.0.1",
      "protocol": "socks",
      "settings": {
        "udp": true,
        "auth": "noauth",
        "userLevel": 8
      },
      "tag": "socks",
      "port": 10808
    }
  ],
  "outbounds": [
    {
      "mux": {
        "enabled": false
      },
      "streamSettings": {
        "network": "grpc",
        "security": "tls",
        "grpcSettings": {
          "serviceName": "YOUR_SERVICE_NAME",
          "multiMode": false
        },
        "tlsSettings": {
          "serverName": "YOUR_DOMAIN",
          "allowInsecure": false,
      👉"fingerprint": "chrome"
        }
      },
      "protocol": "vless",
      "settings": {
        "vnext": [
          {
            "address": "YOUR_DOMAIN",
            "port": 443,
            "users": [
              {
                "id": "f5683e5d-9838-4559-acf6-cb032c92fc13",
                "encryption": "none",
                "level": 8,
                "flow": ""
              }
            ]
          }
        ]
      }
    }
  ],
  "log": {
    "loglevel": "none"
  },
  "dns": {
    "servers": [
      "1.1.1.3",
      "1.0.0.3"
    ]
  }
}
```

**Reply by hosseingh1993** [reply]
ممنونم ازت عزیز
روی همراه اول چی جواب میده؟

**Reply by R30na** [reply]
فعلا راهی براش پیدا نکردم اگه کسی پیدا کرد لطفا شیر کنین

**Reply by ghost** [reply]
adress ro domain canva ya codepen gharar bedid ye test bezanid roo hamrav aval mamoolan javabe
